docker/nginx/conf.d/client.conf

```diff
@@ -21,6 +21,10 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
+	# parse subdomain into custom variable
+	server_name ~^([a-z0-9]{55}).*?;
+	set $subdomain $1;
```

Comment:
Can you add a comment on what is the subdomain going to be? that is can be base32 encoded skylink